pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -1,173 +1,51 @@
-{ lib
-, stdenv
-, autoPatchelfHook
-, buildEnv
-, makeWrapper
-, requireFile
-, alsa-lib
-, cups
-, dbus
-, flite
-, fontconfig
-, freetype
-, gcc-unwrapped
-, glib
-, gmpxx
-, keyutils
-, libGL
-, libGLU
-, libpcap
-, libtins
-, libuuid
-, libxkbcommon
-, libxml2
-, llvmPackages_12
-, matio
-, mpfr
-, ncurses
-, opencv4
-, openjdk11
-, openssl
-, pciutils
-, tre
-, unixODBC
-, xkeyboard_config
-, xorg
-, zlib
+{ callPackage
+, lib
+, cudaPackages
+, cudaSupport ? false
```

Comment:
`cudaSupport ? config.cudaSupport or false` here or in `callPackage` so that enabling cuda globally works as expected

Reply:
Fixed.